### ISSUE TITLE
Coordinates in a more usable form

### ISSUE THREAD

**Issue by jtmarmon** [victorquinn/chancejs#140]
Currently `chance.coordinates()` gives you something like `'12.1237121, 64.123123'`. However, usually coordinates are stored as an array in `[lat, lon]` format. In all of my test cases, I have to write `chance.coordinates().split(', ')`. 

I think this should be the default format, but wanted to get some opinions before I do a PR

**Comment by oliversalzburg** [comment]
You could just use `[chance.latitude(),chance.longitude()]`.

**Comment by jtmarmon** [comment]
yeah, I mean my way of doing it isn't that inconvenient, it just seems like the more natural way to do it. I suppose it's probably not worth a breaking change though

**Comment by victorquinn** [comment]
@jtmarmon I actually agree that it makes more sense to have it as an array though I don't want to break any existing implementations. So let's add it like that, but as an option so it doesn't break things.

I'd also suggest having it (perhaps as an option) `[<longitude>, <latitude>]` to accord with GeoJSON and services such as MapBox, Elasticsearch, and others which (somewhat confusingly) use longitude first, latitude second.

Side note, I have about half implemented a bunch of GeoJSON generators for Chance, hope to push them this weekend, so I'll need this functionality either way :)

**Comment by jtmarmon** [comment]
haha yeah this came to my attention because of geojson related stuff as well
